src/native/fs/fs_napi.cpp

```diff
@@ -2176,7 +2176,6 @@ set_debug_level(const Napi::CallbackInfo& info)
 {
     int level = info[0].As<Napi::Number>();
     DBG_SET_LEVEL(level);
-    LOG("FS::set_debug_level " << level);
```

Comment:
@naveenpaul1 why LOG() prints to console? we have other calls to LOG(), do the other calls be printed to console?
Is there an option to no remove it but print it to the logs instead?

Reply:
@romayalon Other calls not printing to console, 
Two methods are getting called every time config file update, I can try with DBG1()